packages/lib/src/utils/Formatters/types.ts

```diff
@@ -0,0 +1 @@
+export type FormatterFn = (value, context?) => string;
```

Comment:
As you are working on this - do you have more insights about the types of the `value` and `context` objects? It would be great to add here, as I believe they are typed 'any' now

Reply:
I moves this from the Validator file (they were used for the `addressFormatters`, inferring from the code now I think `val` could be a string. Do you know of any more complex use cases for validators where it might need to accept an object? Context should just be an object since it is used to get state from the input field where the formatter is used.

Reply:
I don't recall anymore the possible values of `val` , I just suggested as you are working on this. 

You can add the `string` type and check if there are any linter complains - if not, then we go ahead with that and if in the future turns out there are more types, we fix that.